packages/thumbprint-react/components/Image/index.jsx

```diff
@@ -1,21 +1,33 @@
 import React, { useState, forwardRef, useEffect } from 'react';
+import find from 'lodash/find';
+import classNames from 'classnames';
 import warning from 'warning';
 import PropTypes from 'prop-types';
 import scrollparent from 'scrollparent';
 import { useInView } from 'react-intersection-observer';
-import Picture from './components/picture.jsx';
 import canUseDOM from '../../utils/can-use-dom';
+import styles from './index.module.scss';
+
+// --------------------------------------------------------------------------------------------
+// Steps in rendering Image
+//
+// 1. Picture is rendered without src, srcSets, and with a padding-top placholder on the <img>
+// based on the aspectRatio or containerAspectRatio.
```

Comment:
```suggestion
// based on the containerAspectRatio.
```

Let's just list `containerAspectRatio` since `aspectRatio` isn't a prop in this component?

Reply:
Fix was removed in the reverts.

Reply:
Fixed